cypress/e2e/admin/administrateCtlTools.feature

```diff
@@ -1,101 +1,114 @@
+		Then I see the tool configuration infotext
+		When I select the tool '<ctl_tool_with_optional_param>' from available tools
+		Then I see tool '<ctl_tool_with_optional_param>' is selected
+		When I enter '<param_name>' in optional custom parameter input field schoolParam
+		Then I see custom parameter input field schoolParam contains '<param_name>'
+		When I click on save external tool button
+		Then I see the tool '<ctl_tool_with_optional_param>' in external tools table
 
-        #    Admin deletes tools
-        When I click on administration in menu
-        When I navigate to new school admin page via sub menu
-        When I click on external tools panel
-        Then I see the external tools table
-        Then I see the tool 'CY Test Tool 1' in external tools table
-        Then I see the tool 'CY Test Tool Required Parameters' in external tools table
-        Then I see the tool 'CY Test Tool Optional Parameters' in external tools table
-        When I click on delete button of tool 'CY Test Tool 1'
-        Then I see the external tool deletion dialog title
-        Then I see the external tool deletion dialog information text
-        When I confirm deletion on deletion dialog
-        Then I do not see the tool 'CY Test Tool 1' in external tools table
-        When I click on delete button of tool 'CY Test Tool Required Parameters'
-        Then I see the external tool deletion dialog title
-        Then I see the external tool deletion dialog information text
-        When I confirm deletion on deletion dialog
-        Then I do not see the tool 'CY Test Tool Required Parameters' in external tools table
-        When I click on delete button of tool 'CY Test Tool Optional Parameters'
-        Then I see the external tool deletion dialog title
-        Then I see the external tool deletion dialog information text
-        When I confirm deletion on deletion dialog
-        When I click on delete button of tool 'OpenStreetMap'
-        Then I see the external tool deletion dialog title
-        Then I see the external tool deletion dialog information text
-        When I confirm deletion on deletion dialog
-        Then I do not see the tool 'OpenStreetMap' in external tools table
-        Then I see the external tools table is empty
+		# Admin edits a tool
+		When I click on edit button of tool '<ctl_tool_with_optional_param>'
+		Then I see the external tools configuration page
+		Then I see the external tool configuration page title
+		Then I see the tool configuration infotext
+		Then I see tool '<ctl_tool_with_optional_param>' is selected
+		When I enter '<updated_param_name>' in optional custom parameter input field schoolParam
+		Then I see custom parameter input field schoolParam contains '<updated_param_name>'
+		When I click on save external tool button
+		Then I see the tool '<ctl_tool_with_optional_param>' in external tools table
+		When I click on edit button of tool '<ctl_tool_with_optional_param>'
```

Comment:
from line 73 - 75, repeated the same workflow which is already done above. any reason?